rpm/wpa_supplicant-allow-legacy-renegotiation.patch

```diff
@@ -0,0 +1,20 @@
+diff -up wpa_supplicant-2.10/src/crypto/tls_openssl.c.legacy-server-connect wpa_supplicant-2.10/src/crypto/tls_openssl.c
+--- wpa_supplicant-2.10/src/crypto/tls_openssl.c.legacy-server-connect  2022-01-16 15:51:29.000000000 -0500
++++ wpa_supplicant-2.10/src/crypto/tls_openssl.c        2022-04-28 02:47:26.863529683 -0400
```

Comment:
Also in [Debian](https://sources.debian.org/src/wpa/2%3A2.10-22/debian/patches/allow-legacy-renegotiation.patch/)